### PR TITLE
Support wikidata identifier + fix efg identifications

### PR DESCRIPTION
**Pull request by samleeflang**
https://naturalis.atlassian.net/browse/DD-836

https://naturalis.atlassian.net/browse/DD-721

### CODE REVIEW

File: src/main/java/eu/dissco/core/translator/component/InstitutionNameComponent.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      log.error("Failed to make request to RoR service", e);
+    }
+    log.warn("Could not match name to a ROR id for: {}", url);
+    throw new OrganisationException("Unable to retrieve organisationName");
+  }
+
+  @Cacheable("wikidata")
+  public String getWikiDataName(String wikidata) throws OrganisationException {
+    log.info("Requesting organisation details for organisation: {} with wikidata", wikidata);
+    String url = "https://www.wikidata.org/w/rest.php/wikibase/v0/entities/items/" + wikidata + "/labels/en";
```
Comment (southeo) [inline src/main/java/eu/dissco/core/translator/component/InstitutionNameComponent.java:47]: I don't use append "/labels/en" to the wikidata component in the handle api - should I? Is this just ensuring it returns results in english?

Comment (southeo) [inline src/main/java/eu/dissco/core/translator/component/InstitutionNameComponent.java:47]: I don't use append "/labels/en" to the wikidata component in the handle api - should I? Is this just ensuring it returns results in english?

File: src/test/resources/institution-name/example-wikidata.json [hunk 2]
```diff
@@ -0,0 +1 @@
+"De Museumfabriek"
```
Comment (southeo) [inline src/test/resources/institution-name/example-wikidata.json:1]: Do you need to read this from a file? Why not `MAPPER.readTree(""" ... """)`

**Review by southeo — APPROVED** [review]
Small comments but looks good, preemptively approving :bird:

File: src/main/java/eu/dissco/core/translator/component/InstitutionNameComponent.java (hunk 1 above)
Comment (samleeflang) [reply src/main/java/eu/dissco/core/translator/component/InstitutionNameComponent.java:47]: Yes it doesn't return any additional overhead just the `en` label of wikidata. I think making the assumption/requirement that it needs a `en` label is safe but time will probably tell.

File: src/test/resources/institution-name/example-wikidata.json (hunk 2 above)
Comment (samleeflang) [reply src/test/resources/institution-name/example-wikidata.json:1]: Yes could as well. Tried to keep it in line with the ror but you are right might be better to change

**Pull request merged**